tests/test_batchuploader.py

```diff
@@ -47,19 +48,19 @@ def test_writer_maxitemsize(self):
             ValueTooLarge,
             'Value exceeds max encoded size of 1048576 bytes:'
             ' \'{"b+\\.\\.\\.\'',
-            w.write, {'b'*(m/2): 'x'*(m/2)})
+            w.write, {'b'*int(m/2): 'x'*int(m/2)})
```

Comment:
to be fair, they are not necessary: https://www.python.org/dev/peps/pep-0008/#other-recommendations

pep8 requires one space around comparisons, boolean operators and assignments. other operators are required to have no more than one space and the same amount on both sides. also, quite a shocker for me from `No` section:
```
No:

x = x * 2 - 1
hypot2 = x * x + y * y
c = (a + b) * (a - b)
```